src/rreplay.py

```diff
@@ -175,34 +197,51 @@ def process_messages(subjects):
     None
 
   """
+
+  # retrieve message from rrdump pipe
   subjects_injected = 0
-  message = get_message('rrdump_proc.pipe')
+  message = get_message(consts.RR_PIPE)
+
+  # parse message - retrieve PID, event number, inject state
+  logging.debug("Parsing retrieved message: {}".format(message))
   while message != '':
     parts = message.split(' ')
     inject = parts[0].strip()[:-1]
     event = parts[2]
     pid = parts[4]
+
     # Wait until we can see the process reported by rr to continue
+    logging.debug("Checking if process {} is alive".format(pid))
     while not util.process_is_alive(pid):
       print('waiting...')
     operating = [x for x in subjects if x['event'] == event]
 
     # HACK HACK HACK: we only support spinning off once per event now
     s = operating[0]
+
+    # checking inject state
     if inject == 'INJECT':
+      logging.debug("PID {} is being injected".format(pid))
+
+      # read/write to JSON state file
       with open(s['injected_state_file'], 'r') as d:
         tmp = json.load(d)
       s['pid'] = pid
       tmp['config'] = s
       with open(s['injected_state_file'], 'w') as d:
         json.dump(tmp, d)
 
+      # initiate injector with state file as argument
       s['handle'] = subprocess.Popen(['inject',
                                       s['injected_state_file']])
```

Comment:
The indentation here feels a little weird to me: does this have an extra space before the `s['injected...` or is it just my font?